src/common/models/protocol.enum.ts

```diff
@@ -0,0 +1,17 @@
+export enum Protocol {
```

Comment:
Think **Protocol** --> **LinkType** is better

Reply:
Fixed